samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -129,11 +249,16 @@ static void stream_recv_cb(struct bt_bap_stream *stream,
 		       sink_stream->error_cnt, sink_stream->loss_cnt);
 	}
 }
+#endif /* defined(CONFIG_LIBLC3) */
 
 static struct bt_bap_stream_ops stream_ops = {
 	.started = stream_started_cb,
 	.stopped = stream_stopped_cb,
-	.recv = stream_recv_cb
+#if defined(CONFIG_LIBLC3)
+	.recv = stream_recv_lc3_codec,
+#else
+	.recv = stream_recv_cb,
+#endif /* defined(CONFIG_LIBLC3) */
```

Comment:
Since there is now some expectancy on LC3, we should either verify from the BASE that the broadcaster codec is LC3 when `CONFIG_LIBLC3=y`, or we should do the check in the receive function so that we can decode it if it's LC3 and if not, then we just print it as `stream_recv_cb` currently does